core/src/main/java/nl/nn/adapterframework/pipes/Json2XmlValidator.java

```diff
@@ -393,4 +414,13 @@ public boolean isProduceNamespaceLessXml() {
 		return produceNamespaceLessXml;
 	}
 
+	@IbisDoc({"12", "If true, and converting to or from json, then the message root is the only rootValidation", "true"})
+	public void setValidateJsonToRootElementOnly(boolean validateJsonToRootElementOnly) {
+		this.validateJsonToRootElementOnly = validateJsonToRootElementOnly;
+	}
+	public boolean isValidateJsonToRootElementOnly() {
+		return validateJsonToRootElementOnly;
+	}
```

Comment:
Wat is in deze context de message root validation? Klinkt alsof je 2 root's kan hebben

Reply:
Descender classes kunnen ook nog een soap envelop toevoegen als rootValidatie.